SimpleInventoryManagementSystem/Program.cs

```diff
@@ -1,13 +1,63 @@
 ﻿
+using SimpleInventoryManagementSystem;
 using System;
 
-namespace SimpleInventoryManagementSystem
+namespace SimpleInventoryManagementSystem;
+
+public class Program
 {
-    public class Program
+    enum Services{
+        Exit,
+        AddProduct,
+        ViewProducts
+    }
+
+    public static void PrintServices()
     {
-        public static void Main(string[] args)
+        Console.WriteLine("Choose The number of service you want from our menu bellow");
+        foreach (Services service in Enum.GetValues(typeof(Services)))
         {
-            
+            Console.WriteLine($"{service} is {(int)service}");
+        }
+    }
+    public static void Main(string[] args)
+    {
+        Inventory inventory = new Inventory(new List<Product>());
+
+        while (true)
+        {
+            PrintServices();
+
+            var input = Console.ReadLine();
+            if (!int.TryParse(input,out int number))
+            {
+                throw new ArgumentException("invalid input");
+            }
+
+            switch (number)
+            {
+                case 0:
+                    Console.WriteLine("Goodbye, have a nice day!");
+                    break;
+
+                case 1:
+                    inventory.AddProduct();
+                    break;
+
+                case 2:
+                    inventory.ViewProducts();
+                    break;
+
+                default:
+                    Console.WriteLine("Unfound service, try again");
+                    break;
+            }
```

Comment:
Following up on the comment above, using the enumeration provides us advantages.

1. We won't depend on the underneath value of the Enum. If we decide to give specific numbers to the properties of the Enum, they change, are added or removed, we will have the compiler to help us detect these and change when used.
2. The switch statement will be more readable.

```suggestion
            switch (option)
            {
                case Services.Exit:
                    Console.WriteLine("Goodbye, have a nice day!");
                    return;

                case Services.AddProduct:
                    inventory.AddProduct();
                    break;

                case Services.ViewProducts:
                    inventory.ViewProducts();
                    break;

                default:
                    Console.WriteLine("Unfound service, try again");
                    break;
            }
```